lib/resources/dstu2/document_reference_docref.yaml

```diff
@@ -0,0 +1,33 @@
+---
```

Comment:
I think the convention has been to only include the values in these configs which will be reflected in the output. Since this is being used with for the terminology bindings, there's some fields in here that can be removed